templates/master/00-master/baremetal/files/baremetal-keepalived-keepalived.yaml

```diff
@@ -23,6 +23,12 @@ contents:
         weight 50
     }
 
+    {{`{{if .Cluster.APIProvisioningVIP}}`}}
+    vrrp_script chk_api_prov {
+        script "/usr/bin/curl -o /dev/null -kLs https://{{`{{ .Cluster.APIProvisioningVIP }}`}}:2379"
```

Comment:
Shouldn't you check here the local ('localhost:2379') service status?

Reply:
Done